blocks/api/paste/strip-attributes.js

```diff
@@ -0,0 +1,12 @@
+export default function( nodes ) {
+	// MUTATION!
+	nodes.forEach( deepAttributeStrip );
+	return nodes;
+}
+
+function deepAttributeStrip( node ) {
+	node.removeAttribute( 'style' );
+	node.removeAttribute( 'class' );
+	node.removeAttribute( 'id' );
+	Array.from( node.children ).forEach( deepAttributeStrip );
```

Comment:
This is just stripping it from the parent, right? That probably won't be enough. We need to strip it from all elements.